src/css/profile/mobile/common/bottombar.less

```diff
@@ -1,17 +1,29 @@
 .ui-footer {
-    height: 56 * @px_base;
     padding: 0;
     .ui-bottom-bar {
         display: flex;
         width: 100%;
         min-height: 56 * @px_base;
         background-color: var(--bottom-bar-color);
-        margin: 0 24 * @px_base;
+        padding: 0 24 * @px_base;
     
         .ui-btn {
             font-size: 18 * @sp_base;
             max-width: 100%;
             height: 56 * @px_base;
+            &.ui-btn ~ .ui-btn {
+                margin-left: 0;
+            }
+            &.ui-btn-icon {
+                font-size: 12 * @sp_base;
```

Comment:
sohuln't be @px_base ?

Reply:
This is only for font size... just in case if we want to change something.

Reply:
@px_base is relevant of dp unit
@sp_base is relevant to sp unit
Both @px_base and @sp_base are defined as 1px now but this can be change in the future and these units can be different.